python/mujoco/rollout.cc

```diff
@@ -188,15 +188,20 @@ PYBIND11_MODULE(_rollout, pymodule) {
   // get subsequent states and corresponding sensor values
   pymodule.def(
       "rollout",
-      [](const MjModelWrapper& m, MjDataWrapper& d,
+      [](py::list m, MjDataWrapper& d,
          int nstep, unsigned int control_spec,
          const PyCArray state0,
          std::optional<const PyCArray> warmstart0,
          std::optional<const PyCArray> control,
          std::optional<const PyCArray> state,
          std::optional<const PyCArray> sensordata
          ) {
-        const raw::MjModel* model = m.get();
+        // get raw pointers
+        int nroll = state0.shape(0);
+        const raw::MjModel* model_ptrs[nroll];
```

Comment:
Hold on, you can't do this, this would be a variable length array on the stack.

Reply:
Saran says: use std::vector here

Reply:
fixed